docs/creating_integration_manifest.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Integration Manifest"
+sidebar_label: "Manifest"
+---
+
+Each integration has a manifest file to specify basic information about an integration. This file is stored as `manifest.json` in your integration directory. It is required to add such a file, including for custom components.
+
+```json
+{
+  "domain": "hue",
+  "name": "Philips Hue",
+  "documentation": "https://www.home-assistant.io/components/hue",
+  "dependencies": ["mqtt"],
+  "codeowners": ["@balloob"],
+  "requirements": ["aiohue==1.9.1"]
+}
+```
+
+Or a minimal example that you can copy into your project:
+
+```json
+{
+  "domain": "your_domain_name",
+  "name": "Your Intgration",
+  "documentation": "https://www.example.com",
+  "dependencies": [],
+  "codeowners": [],
+  "requirements": []
+}
+```
+
+## Domain
+
+The domain is a short name consisting of characters and underscores. This domain has to be unique and cannot be changed. Example of the domain for the mobile app integration: `mobile_app`.
+
+## Name
+
+The name of thet integration.
+
+## Documentation
+
+Website that contains documentation on how to use your integration. If this integration is being submitted for inclusion in Home Assistant, it should be `https://www.home-assistant.io/components/<domain>`
```

Comment:
```suggestion
Website that contains documentation on how to use your integration. If this integration is being submitted for inclusion in Home Assistant, it should usually be `https://www.home-assistant.io/components/<domain>`
```

Reply:
I disagree. If the integration is going to be part of Home Assistant, it should always have that format .

Reply:
Well as an example, the iOS component pages are pretty much bare and redirect people to the iOS app docs which are seperate. Maybe that's an edge case.

Reply:
That is an edge case.